merge_requests.go

```diff
@@ -511,6 +511,58 @@ func (s *MergeRequestsService) CancelMergeWhenPipelineSucceeds(pid interface{},
 	return m, resp, err
 }
 
+// Subscribe subscribes the authenticated user to the given merge request to receive notifications.
+// If the user is already subscribed to the merge request, the status code 304 is returned.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/merge_requests.html#subscribe-to-a-merge-request
```

Comment:
Could you please keep the length of comments to ~80 chars? The URL is sometimes longer, but I try to keep the sentences to max ~80 chars.

FYI, while I'm sharing coding style preferences, for lines of code I try to limit the length to ~100 chars :wink: